spec/timezone.html

```diff
@@ -3,5 +3,240 @@
 
 <emu-clause id="sec-temporal-timezone">
   <h1>Temporal.TimeZone Objects</h1>
-  <p><mark>TODO: Specify.</mark></p>
+  <p>A Temporal.TimeZone object is an immutable Object referencing a time zone.</p>
+
+  <emu-clause id="sec-temporal-timezone-constructor">
+    <h1>The Temporal.TimeZone Constructor</h1>
+    <p>
+      The Temporal.TimeZone constructor is the %Temporal.TimeZone% intrinsic object.
+      When called as a constructor, it creates and initializes a new Temporal.TimeZone object.
+    </p>
+    <p>
+      The Temporal.TimeZone constructor is designed to be subclassable.
+      It may be used as the value of an extends clause of a class definition.
+      Subclass constructors that intend to inherit the specified TimeZone behaviour must include a super call to the %Temporal.TimeZone% constructor to create and initialize subclass instances with the necessary internal slots.
+    </p>
+
+    <emu-clause id="sec-temporal.timezone">
+      <h1>Temporal.TimeZone ( _identifier_ )</h1>
+      <p>
+        When the `Temporal.TimeZone` function is called, the following steps are taken:
+      </p>
+      <emu-alg>
+        1. If NewTarget is *undefined*, then
+          1. Throw a *TypeError* exception.
+        1. Let _identifier_ be ? ToString(_identifier_).
+        1. If _identifier_ matches `/([+-][01]?[0-9]):?([0-5][0-9])/`, then
+          1. <mark>Do something with the offset?</mark>
+        1. If ! IsValidTimeZoneName(_identifier_) is *false*, then
+          1. Throw a *TypeError* exception.
+        1. Let _canonical_ be ! CanonicalizeTimeZoneName(_identifier_).
+        1. Return ? CreateTimeZone(_canonical_, NewTarget).
+      </emu-alg>
+    </emu-clause>
+  </emu-clause>
+
+  <emu-clause id="sec-properties-of-the-temporal-timezone-constructor">
+    <h1>Properties of the Temporal.TimeZone Constructor</h1>
+    <p>The value of the [[Prototype]] internal slot of the Temporal.TimeZone constructor is the intrinsic object %FunctionPrototype%.</p>
+    <p>The Temporal.TimeZone constructor has the following properties:</p>
+
+    <emu-clause id="sec-temporal-timezone-prototype">
+      <h1>Temporal.TimeZone.prototype</h1>
+      <p>The initial value of Temporal.TimeZone.prototype is %Temporal.TimeZone.prototype%.</p>
+      <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal.timezone.prototype-@@iterator">
+      <h1>Temporal.TimeZone[ @@iterator ]</h1>
+      <p>
+        <mark>TODO</mark>
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal.timezone.from">
+      <h1>Temporal.TimeZone.from ( _arg_ )</h1>
+      <p>
+        The `from` method takes one argument _arg_.
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Return ? ToTimeZone(_arg_).
```

Comment:
reminder for #239 / #196

Reply:
Is this inline with the polyfill though? I got confused here since I'm not sure if someone would subclass `TimeZone` but honestly that shouldn't matter given that we mention

>       The Temporal.TimeZone constructor is designed to be subclassable.

above.